src/styles/variables.less

```diff
@@ -25,12 +40,8 @@
 @color-warning: #feb209;
 @color-danger: #f7403a;
 
-@color-white: #fff;
-
-@color-gray-5: fade(#000, 5%);
-@color-gray-10: fade(#000, 10%);
-@color-gray-25: fade(#000, 25%);
-@color-gray-30: fade(#000, 30%);
+@color-borderColor: @color-gray;
```

Comment:
`default` is related to the default button coloring.  I would think that we would use `@color-borderColor` anytime we need to define a border we would set the color using this variable